src/conf_macros.m4

```diff
@@ -651,6 +651,23 @@ AC_DEFUN([WITH_FILES_PROVIDER],
     AM_CONDITIONAL([BUILD_FILES_PROVIDER], [test x"$with_files_provider" = xyes])
   ])
 
+AC_DEFUN([WITH_EXTENDED_ENUMERATION_SUPPORT],
+  [ AC_ARG_WITH([extended-enumeration-support],
+                [AC_HELP_STRING([--with-extended-enumeration-support],
+                                [Whether to build enumeration support for
+                                 IPA and AD providers[no].]
```

Comment:
Thanks for the PR. It looks good to me. Just one nitpicking comment from typography point of view -- there should be space between `provides` and `[no]` (as we have it in other conditions too).

Reply:
Thank you, fixed.